common/src/main/kotlin/org/valkyrienskies/mod/common/PlayerUtil.kt

```diff
@@ -45,9 +45,9 @@ object PlayerUtil {
             val position = shipMatrix.transformPosition(
                 player.position().toJOML()
             )
-            val yaw = atan2(direction.x, -direction.z) // yaw in radians
-            val pitch = asin(-direction.y)
-            player.yRot = (yaw * (180 / Math.PI)).toFloat() + 180
+            val yaw = -atan2(direction.x, direction.z)
+            val pitch = -atan2(direction.y, sqrt((direction.x * direction.x) + (direction.z * direction.z)))
```

Comment:
Pitch is always between -90 and 90, so I don't think atan2 is necessarily what we want here, because it can return numbers outside of that range.

Reply:
It was doing wierd until i used this so :person_shrugging: